NativeCharList/Assets/NativeString.cs

```diff
@@ -247,10 +437,14 @@ public static void OverWrite(ref NativeString A, int position, NativeString stam
             }
         }
 
```

Comment:
I think on Friday I will write some wrappers so that it is trivial to write replace and insert algorithms without having to allocate new NativeStrings. In the meantime I suggest you focus on getting Append() methods for the different data types (int, float, double, char, byte, short, string, NativeString, ect. And then make + operator overloads for them.